src/components/CommentCard.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable */
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export default function CommentCard() {
+  const { t } = useTranslation();
+
+  return (
+    <div className="bg-white text-sm md:text-base dark:bg-dark-bg w-full border rounded-md border-gray-300 dark:border-white my-5">
+      <div className="flex flex-row p-5 w-full">
+        <div className="w-1/4 md:w-1/6 mr-2">
+          <img
+            className="w-10 h-10 md:w-20 md:h-20 rounded-full"
+            src="https://cdn.pixabay.com/photo/2019/11/11/04/33/child-4617142_1280.jpg"
+            alt="Profile"
+          />
+        </div>
+        <div className="flex flex-col">
+          <div className="flex flex-row">
+            <div className="mr-2">Kenneth Ngabo</div>
+            <div className="text-gray-400">{t('Wednesday')}, 16:21</div>
+          </div>
+          <div className="my-2">
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim veniam
+            </p>
+          </div>
+          <div className="flex flex-row">
+            <div className="flex flex-row">
+              <img
+                className="relative rounded-full w-7 h-7"
+                src="https://cdn.pixabay.com/photo/2018/02/16/14/38/portrait-3157821_1280.jpg"
+                alt="Comment profile"
+              />
+              <img
+                className="absolute ml-4 rounded-full w-7 h-7"
+                src="https://cdn.pixabay.com/photo/2016/11/22/21/42/woman-1850703_1280.jpg"
+                alt="Comment profile"
+              />
+            </div>
+            <div className="ml-6 text-primary cursor-pointer">
+              <h3>4 {t('Replies')}</h3>
+            </div>
+            <div className="ml-2 text-gray-400">
+              <h3>{t('Last reply wednesday, 15th')}</h3>
```

Comment:
This will be dynamic data that will be translated from the backend whenever possible. When switching to using data passed to the comment, you will not need to translate this field.

Reply:
Solved.